pcsx2/gui/MainFrame.cpp

```diff
@@ -450,8 +450,8 @@ void MainEmuFrame::CreateWindowsMenu()
 void MainEmuFrame::CreateCaptureMenu()
 {
 	m_menuCapture.Append(MenuId_Capture_Video, _("Video"), &m_submenuVideoCapture);
-	m_submenuVideoCapture.Append(MenuId_Capture_Video_Record, _("Start Recording"));
-	m_submenuVideoCapture.Append(MenuId_Capture_Video_Stop, _("Stop Recording"))->Enable(false);
+	m_submenuVideoCapture.Append(MenuId_Capture_Video_Record, _("Start screenrecorder (Video with audio + audio-only)"));
```

Comment:
There was a small miscommunication here. The video file does come with audio embedded in it. AVI - video only; WAV - audio only.
For now, just saying "Start screenrecorder" is probably sufficient.